packages/react/src/server/richtext.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import StoryblokServerComponent from './server-component';
```

Comment:
non-blocking: Probably not part of the scope of this PR

@imranolas do we have separate richtext files because one uses `StoryblokComponent` and the other `StoryblokServerComponent` right? That would affect only the componentResolver method. Any chance we can avoid having duplicate code for the rest? How would you structure it?